db/wide/wide_columns_helper.cc

```diff
@@ -0,0 +1,33 @@
+//  Copyright (c) Meta Platforms, Inc. and affiliates.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#include "db/wide/wide_columns_helper.h"
+
+#include "db/wide/wide_column_serialization.h"
+
+namespace ROCKSDB_NAMESPACE {
+void WideColumnsHelper::DumpWideColumns(const WideColumns& columns,
+                                        std::ostream& oss, bool hex) {
```

Comment:
Very minor but we might want to rename this parameter to `os` for `ostream` (`oss` stands for `ostringstream`); same below